modules/user/server-ts/social/googleExpo/resolvers.js

```diff
@@ -0,0 +1,51 @@
+import { access } from '@gqlapp/authentication-server-ts';
+
+import User from '../../sql';
+
+const registerUser = async (id, email) => {
+  return User.register({
+    username: email,
+    email: email,
+    password: id,
+    isActive: true
+  });
+};
+
+const createGoogleOAuth = async user => User.createGoogleOAuth(user);
+
+export default () => ({
+  Mutation: {
+    async googleExpoLogin(
```

Comment:
Very bad approach. Every client is able to send a request to the server passing any `id | name | email` and successfully being registered in the system. Without any verifications. User **must** be authenticated on the server side.

Reply:
done